### ISSUE TITLE
Fails to build on macOS

### ISSUE THREAD

**Issue by dit7ya** [houmain/keymapper#106]
The latest release fails to build on macOS while the previous version (3.5.1) builds with some warnings.

```fish
[ 61%] Building CXX object CMakeFiles/keymapperd.dir/src/server/unix/GrabbedDevicesMacOS.cpp.o
[ 65%] Building CXX object CMakeFiles/keymapperd.dir/src/server/unix/main.cpp.o
[ 69%] Building CXX object CMakeFiles/keymapperd.dir/src/common/output.cpp.o
[ 73%] Linking CXX executable keymapperd
ld: Undefined symbols:
  GrabbedDevices::update_devices(), referenced from:
      _main in main.cpp.o
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [CMakeFiles/keymapperd.dir/build.make:241: keymapperd] Error 1
make[1]: *** [CMakeFiles/Makefile2:111: CMakeFiles/keymapperd.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

**Comment by dit7ya** [comment]
Also, I am trying to get `3.5.1` to work. I have karabiner-elements `14.13.0` installed but I am getting this error.

```
ERROR: Karabiner driver version mismatched
ERROR: Creating virtual device failed
```

**Comment by houmain** [comment]
Thanks for reporting! This shoud be fixed with the 3.5.3 release.
